src/core/net/ip6.cpp

```diff
@@ -184,6 +187,8 @@ ThreadError Ip6::InsertMplOption(Message &aMessage, Header &aIp6Header, MessageI
 {
     ThreadError error = kThreadError_None;
 
+    otLogFuncEntry();
```

Comment:
I would also prefer that we do not add these logs into the default code base, but rather manually add them as necessary.  I would be less adverse to these logs if they could be toggled per module, alas that is not the case as it is all or nothing.  If I am interested in seeing logs for one or two functions in a particular module, I don't want to have to enable these logs across the entire library.